docs/examples/tabs/basic.vue

```diff
@@ -8,10 +8,11 @@
 </template>
 <script lang="ts" setup>
 import { ref } from 'vue'
+import { TabsPaneContext } from 'element-plus'
```

Comment:
Here should be import as type
```suggestion
import type { TabsPaneContext } from 'element-plus'
```

Reply:
- you are right. I fixed this problem.